libraries/DataFlash/LogStructure.h

```diff
@@ -818,7 +818,8 @@ struct PACKED log_GPS_SBF_EVENT {
 
 struct PACKED log_Esc {
     LOG_PACKET_HEADER;
-    uint64_t time_us;     
+    uint64_t time_us; 
+    uint8_t esc_id;
```

Comment:
That's nice, except it will break people's graphing a log. There is no way in MP (and I think APM Planner is the same) to graph stuff separated by a field in a message. This is why we have multiple message types for many things.

Reply:
Is 4-8 different type of messages just for different ESCs can be considered an overkill?